scripts/make-dist-v2.ts

```diff
@@ -1,8 +1,8 @@
-import path from 'path';
-import fs from 'fs';
 import { Command } from 'commander';
 import { parse as CommentJsonParse } from 'comment-json';
-import { CodePackman } from 'autoscale-core';
+import fs from 'fs';
+import path from 'path';
+import { CodePackman } from './code-packman';
```

Comment:
do we now have 2 copies of code-packman.ts ? one in each project?

maybe it should be published similar to ftnt-ci tools?

Reply:
the script code-packman is still experimental and only used by the fortigate-autoscale-aws. I still don't find it reasonable to make it a tool for now. The same script in the autoscale-core isn't used at this moment but I'd like to remove it from the autoscale-core.